calendar/quickstart/quickstart.rb

```diff
@@ -0,0 +1,57 @@
+require "google/apis/calendar_v3"
```

Comment:
Not sure if we need this after speaking with Dione

Reply:
agreed. it seems like we are overcomplicating things